recipes/pyferret/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+export LIBZ_DIR=$PREFIX
+export READLINE_DIR=$PREFIX
+export HDF5_DIR=$PREFIX
+export NETCDF4_DIR=$PREFIX
+export FC="gfortran"
+export LD_X11="-L/usr/lib64 -lX11"
+
+export OS=$(echo $(uname) | tr '[:upper:]' '[:lower:]')
+export PLAT=$(uname -p)
+export BUILDTYPE=$PLAT-$OS
+export HOSTTYPE=$BUILDTYPE
+export FER_DIR=$PREFIX
+
+make
+# make run_tests
```

Comment:
It is not unusual for the the images to not match exactly.  We check any
differences by hand (unfortunately) using something like the compare
program to see that there are no real differences, or that the differences
are not unexpected (such as difference in fonts on some of the plots).  I
suspect I need to figure something out to make images that are more stable.

Thanks,
Karl



On Wed, Jun 22, 2016 at 9:19 PM, Filipe <notifications@github.com> wrote:

> In recipes/pyferret/build.sh
> <https://github.com/conda-forge/staged-recipes/pull/810#discussion_r68166832>
> :
>
> > +
> > +export LIBZ_DIR=$PREFIX
> > +export READLINE_DIR=$PREFIX
> > +export HDF5_DIR=$PREFIX
> > +export NETCDF4_DIR=$PREFIX
> > +export FC="gfortran"
> > +export LD_X11="-L/usr/lib64 -lX11"
> > +
> > +export OS=$(echo $(uname) | tr '[:upper:]' '[:lower:]')
> > +export PLAT=$(uname -p)
> > +export BUILDTYPE=$PLAT-$OS
> > +export HOSTTYPE=$BUILDTYPE
> > +export FER_DIR=$PREFIX
> > +
> > +make
> > +# make run_tests
>
> The tests are returning a bunch of mismatch images, but they look "OK" to
> me. Maybe that is a false positive?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/810/files/fa92c5fe78498937635aa143e33fb9920d92acca#r68166832>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/APpYx31PgBN4Z_mxVbxg-dqTtgFkLYzUks5qOe0mgaJpZM4I4j_0>
> .
>



-- 
Karl M. Smith, Ph.D.
JISAO Univ. Wash. and PMEL NOAA
"The contents of this message are mine personally and do
not necessarily reflect any position of the Government
or the National Oceanic and Atmospheric Administration."